docs/make.jl

```diff
@@ -58,11 +27,10 @@ makedocs(;
     pages=[
         "Home" => "index.md",
         "userguide.md",
-        "API" => ["api/index.md", "api/sparsevariational.md", "api/laplace.md"],
-        "Examples" =>
-            map(filter!(filename -> endswith(filename, ".md"), readdir(EXAMPLES_OUT))) do x
-                return joinpath("examples", x)
-            end,
+        "API" => joinpath.(Ref("api"), ["index.md", "sparsevariational.md", "laplace.md"]),
+        "Examples" => map(filter!(isdir, readdir("examples"))) do x
+            joinpath("examples", x, "example.md")
```

Comment:
why this change ? it seems a bit weird to have them all be called "example" ... :confused: